src/ops/constant.rs

```diff
@@ -450,15 +446,15 @@ mod test {
         val.check_type(&ClassicType::Container(Container::Opaque(typ_int.clone())))
             .unwrap();
 
-        let typ_float = CustomType::new(
+        let typ_qb = CustomType::new(
             "mytype",
-            vec![TypeArg::ClassicType(ClassicType::F64)],
+            vec![TypeArg::Type(SimpleType::Qubit)],
```

Comment:
There is something suspicious here in that the tag passed below is `TypeTag::Hashable`....

Reply:
any reason the custom type can't be hashable with a qubit as type argument?